web/src/components/SecureAdminConsole.jsx

```diff
@@ -34,10 +34,13 @@ class SecureAdminConsole extends React.Component {
         }
 
         const apps = await this.props.onLoginSuccess();
+        const pendingApps = await this.props.pendingApp();
         this.setState({ authLoading: false });
         if (apps.length > 0) {
           this.props.history.replace(`/app/${apps[0].slug}`);
-        } else {
+        }else if (pendingApps?.slug) {
```

Comment:
again, it's a single app, not apps, so the name `pendingApps` is misleading here